swift-sdk/Internal/EmbeddedSessionManager.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  EmbeddedSessionManager.swift
+//  
+//
+//  Created by Andrew Nelson on 5/30/23.
+//
+
+import Foundation
+
+public class EmbeddedSessionManager {
+    public static let shared = EmbeddedSessionManager()
+    public var currentSession: IterableEmbeddedSession?
```

Comment:
`session` instead of `currentSession` as there we have only one session one time and no different states of session?